forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -3032,7 +3032,9 @@ public static final String applyTextChangeEffects(final String def, final boolea
         }
```

Comment:
because the ChangedWords only use uppercase now we need to add lowercase version above like Any already has
for types it should be fine because they're all uppercase anyway?

but that alone isn't enough, it looks like `changedCardKeywordsByWord` isn't used anywhere

Reply:
i fixed color word replacement above

Reply:
it will modify the KW now, but it's still not calculating the new ones

Reply:
it seems to have been broken for a bit longer?
changedCardKeywordsByWord needs to be added to getChangedCardKeywordsList